docker/dev/Dockerfile

```diff
@@ -37,42 +36,8 @@ USER www-data
 # Build the farmOS codebase in /var/farmOS.
 RUN /usr/local/bin/build-farmOS.sh
 
-# Configure PHP CodeSniffer.
-RUN { \
-    echo '<?xml version="1.0" encoding="UTF-8"?>'; \
-    echo '<ruleset name="farmOS">'; \
-    echo '  <description>PHP CodeSniffer configuration for farmOS development.</description>'; \
-    echo '  <file>.</file>'; \
-    echo '  <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,yml"/>'; \
-    echo '  <config name="drupal_core_version" value="9"/>'; \
-    echo '  <rule ref="Drupal">'; \
-    echo '    <exclude name="Drupal.InfoFiles.AutoAddedKeys.Project"/>'; \
-    echo '    <exclude name="Drupal.InfoFiles.AutoAddedKeys.Version"/>'; \
-    echo '    <exclude name="Drupal.Arrays.Array.LongLineDeclaration"/>'; \
-    # @todo https://www.drupal.org/project/coder/issues/2159253
-    echo '    <exclude name="Drupal.Commenting.InlineComment.SpacingAfter"/>'; \
-    echo '  </rule>'; \
-    echo '  <rule ref="DrupalPractice">'; \
-    # @todo https://www.drupal.org/project/coder/issues/2159253
-    echo '    <exclude name="DrupalPractice.Commenting.CommentEmptyLine.SpacingAfter"/>'; \
-    echo '  </rule>'; \
-    echo '  <rule ref="PHPCompatibility"/>'; \
-    echo '  <config name="testVersion" value="7.4-"/>'; \
-    echo '  <rule ref="Internal.Tokenizer.Exception"><severity>0</severity></rule>'; \
-    echo '</ruleset>'; \
-  } > /var/farmOS/phpcs.xml
-
-# Configure PHPStan.
-RUN { \
-    echo 'parameters:'; \
-    echo '  level: 1'; \
-    echo '  ignoreErrors:'; \
-    echo '    # new static() is a best practice in Drupal, so we cannot fix that.'; \
-    echo "    - '#^Unsafe usage of new static#'"; \
-    echo '  scanDirectories:'; \
-    echo '    # Tell PHPStan where to find GeoPHP classes.'; \
-    echo '    - vendor/itamair/geophp'; \
-  } > /var/farmOS/phpstan.neon
+# Add Configurartions for PHP CodeSniffer, PHPStan.
+COPY --chown=www-data ./files/ /var/farmOS/
 
 # Configure PHPUnit.
 RUN cp -p /var/farmOS/web/core/phpunit.xml.dist /var/farmOS/phpunit.xml \
```

Comment:
Could we move the PHPunit config to a separate file in `files/phpunit.xml` ? I know we're copying this over from the Drupal core `phpunit.xml.dist` but I wonder how often this changes. We haven't needed to modify our code in a few years.. but I wonder how much that is just luck not having conflicting changes from Drupal core :thinking:

Reply:
@AlMaVizca did separate this one out originally and I asked that we keep it the way it was, because separating it out means we are maintaining the whole file. I agree it may not change often, but if it does there's no way for us to know. Currently we get any changes automatically (or the `sed` command will fail, notifying us that we need to update it). I think that's worth keeping as-is.

Reply:
@paul121 @mstenta you will see on a following PR that, I'v created the script with all the regexp, so the Docker file is cleaner